js/huobipro.js

```diff
@@ -448,13 +450,16 @@ module.exports = class huobipro extends Exchange {
     }
 
     async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
-        await this.loadMarkets ();
-        let response = await this.privateGetOrderMatchresults (params);
-        let trades = this.parseTrades (response['data'], undefined, since, limit);
-        if (symbol !== undefined) {
-            let market = this.market (symbol);
-            trades = this.filterBySymbol (trades, market['symbol']);
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchMyTrades() requires a symbol argument');
```

Comment:
Oops, i've just spotted this line too. I'm pleased to inform you that despite of [huobi documentation](https://github.com/huobiapi/API_Docs_en/wiki/REST_Reference#get-v1ordermatchresults----get-order-matchresults) suggests the opposite, the `symbol` argument is not mandatory for `this.privateGetOrderMatchresults` call (you can test it for yourself). Therefore i'm not sure we should throw `ArgumentsRequired` error here.